src/actions/registration-invitation-actions.js

```diff
@@ -141,8 +122,7 @@ export const exportInvitationsCSV = (term, order, orderDir, filters = {}) => asy
     const accessToken = await getAccessTokenSafely();
     const { currentSummit }   = currentSummitState;
     const filename = currentSummit.name + '-invitations.csv';
-    const {tagFilter, isAccepted, isSent, allowedTicketTypesIds} = filters;
-    const filter = [];
+    const filter = parseFilters(filters);
```

Comment:
add term inside parse filter
otherwise would not be properly parsed with or() or and()